ci/scripts/github/common.sh

```diff
@@ -72,8 +72,8 @@ function update_conda_env() {
     # Deactivate the environment first before updating
     conda deactivate
 
-    # Update the packages with --prune to remove any extra packages
-    rapids-mamba-retry env update -n morpheus --prune -q --file ${MORPHEUS_ROOT}/docker/conda/environments/cuda${CUDA_VER}_dev.yml
+    # Update the packages
+    rapids-mamba-retry env update -n morpheus -q --file ${MORPHEUS_ROOT}/docker/conda/environments/cuda${CUDA_VER}_dev.yml
```

Comment:
@mdemoret-nv The problem is for this pr we now have two yaml files that are installed in the CI runner:
```
CONDA_ALWAYS_YES=true /opt/conda/bin/mamba env create -n ${PROJ_NAME} -q --file /tmp/conda/cuda${CUDA_SHORT_VER}_dev.yml
CONDA_ALWAYS_YES=true /opt/conda/bin/mamba env update -n ${PROJ_NAME} -q --file /tmp/conda/cuda${CUDA_SHORT_VER}_examples.yml && \
```

The problem, if I understand correctly, then is if we use `--prune` in `test.sh` it will effectively uninstall dask, cuml and tensorflow.

The alternative would be to update `cuda_11.8_examples.yml` to include everything currently in `cuda11.8_dev.yml`

Reply:
Ya we are effectively doing the same thing in MRC. In that PR, I use `conda-merge` to merge all of the environment files into a single file and then you can call `--prune` on it